packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -569,8 +594,11 @@ String _createPluginClassName(String name) {
 
 String _createUTIIdentifier(String organization, String name) {
   // Create a UTI (https://en.wikipedia.org/wiki/Uniform_Type_Identifier) from a base name
+  name = camelCase(name);
   final RegExp disallowed = RegExp(r'[^a-zA-Z0-9\-\.\u0080-\uffff]+');
-  name = camelCase(name).replaceAll(disallowed, '');
+  organization = organization.replaceAll(disallowed, '');
+  organization = organization.isEmpty ? 'untitled' : organization;
+  name = name.replaceAll(disallowed, '');
```

Comment:
You could just call `replaceAll` once on the merged value that is returned below, instead of calling it for each part.